wagtail/admin/templates/wagtailadmin/pages/confirm_delete.html

```diff
@@ -9,11 +9,24 @@
     <div class="nice-padding">
         <p>
             {% trans 'Are you sure you want to delete this page?' %}
-            {% if descendant_count %}
-                {% blocktrans count counter=descendant_count %}
-                    This will also delete one more subpage.
+
+            {% if descendant_count and translation_count %}
+                {% blocktrans trimmed count counter=translation_count %}
+                    Deleting this page will also delete {{ descendant_count }} more child pages, {{ translation_count }} translation of this page and their combined {{ combined_subpages }} translated child pages.
```

Comment:
heh, looking at this in practice it _almost_ works. if `descendant_count == 1`, this reads "Deleting this page will also delete 1 more child pages" which is a bit wrong.

```jinja
{% if descendant_count %} {# has descendants #}
    {% blocktrans trimmed count counter=descendant_count %}
    Deleting this page will also delete {{ descendant_count }} child page.
    {% plural %}
    Deleting this page will also delete {{ descendant_count }} more child pages.
    {% endblocktrans %}

    {% if translation_count %} {# has translations #}
        {% if translation_descendant_count %} {# has translations with descendants #}
            {% if translation_count == 1 %}
                {% blocktrans trimmed count counter=translation_descendant_count %}
                It will also delete 1 translation and its combined {{ translation_descendant_count }} translated child page.
                {% plural %}
                It will also delete 1 translation and its combined {{ translation_descendant_count }} translated child pages.
                {% endblocktrans %}
            {% else %}
                {% blocktrans trimmed count counter=translation_descendant_count %}
                It will also delete {{ translation_count }} translations and their combined {{ translation_descendant_count }} translated child page.
                {% plural %}
                It will also delete {{ translation_count }} translations and their combined {{ translation_descendant_count }} translated child pages.
                {% endblocktrans %}
            {% endif %}
        {% else %} {# has translations but no descendants #}
            {% blocktrans trimmed count counter=translation_count %}
            It will also delete {{ translation_count }} translation.
            {% plural %}
            It will also delete {{ translation_count }} translations.
            child pages.
            {% endblocktrans %}
        {% endif %}
    {% endif %}
{% elif translation_count %} {# no descendants #}
    {% if translation_descendant_count %} {# has translations with descendants #}
        {% if translation_count == 1 %}
            {% blocktrans trimmed count counter=translation_descendant_count %}
            Deleting this page will also delete 1 translation and its combined {{ translation_descendant_count }} translated child page.
            {% plural %}
            Deleting this page will also delete 1 translation and its combined {{ translation_descendant_count }} translated child pages.
            {% endblocktrans %}
        {% else %}
            {% blocktrans trimmed count counter=translation_descendant_count %}
            Deleting this page will also delete {{ translation_count }} translations and their combined {{ translation_descendant_count }}
            translated child page.
            {% plural %}
            Deleting this page will also delete {{ translation_count }} translations and their combined {{ translation_descendant_count }}
            translated child pages.
            {% endblocktrans %}
        {% endif %}
    {% else %}
        {% blocktrans trimmed count counter=translation_count %}
        Deleting this page will also delete {{ translation_count }} translation of this page.
        {% plural %}
        This will also delete {{ descendant_count }} more subpages.
        Deleting this page will also delete {{ translation_count }} translations of this page.
        {% endblocktrans %}
    {% endif %}
{% endif %}
```

is probably closer to the truth (need verifying). I split the component parts into two block as the number of variations was too much. cc @phildexter